# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../jpilot/lib64.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2006-11-09 13:42:14.000000000 +0200
+++ ./configure.in	2006-11-09 13:42:56.000000000 +0200
@@ -135,7 +135,7 @@
 PILOT_LIBS="-lpisock"
 
 if test $pilotlibs = no ; then
-for pilot_libs in $pilot_prefix/lib /usr/lib /usr/local/lib/ \
+for pilot_libs in $pilot_prefix/lib{,64} /usr/lib{,64} /usr/local/lib/ \
   /usr/extra/pilot/lib ; do
    if test -r $pilot_libs/libpisock.so >/dev/null 2>&1 ; then
       pilotlibs=yes
